test/integration/targets/git/vars/main.yml

```
@@ -33,6 +33,7 @@ separate_git_dir: '{{ remote_tmp_dir }}/sep_git_dir'
 repo_format1: 'https://github.com/jimi-c/test_role'
 repo_format2: 'git@github.com:jimi-c/test_role.git'
 repo_format3: 'ssh://git@github.com/jimi-c/test_role.git'
+repo_format4: 'username:password@https://github.com/thisdoesnotexists/test_role'
```

```suggestion
repo_format4: 'https://username:password@github.com/thisdoesnotexists/test_role'
```

I suppose the former works, but it's not a real format :)

---

Real format does not raise an error in `stderr` like - 

```
git clone username:password@https://example_website.com
Cloning into 'example_website.com'...
fatal: protocol 'username:password@https' is not supported
```

---

Oh, I see. Add a comment then please, so that no one else makes the same mistake I did. I thought it was an accident, but it's a purposeful errant format.